iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveySelectionView.swift

```diff
@@ -13,6 +13,8 @@ struct SurveySelectionView: View {
 
     @StateObject private var dataSource = DataSource()
 
+    @Binding var showingAccountView: Bool
```

Comment:
@blyscuit We usually use checking prefix like: `isShowingAccountView`. Should we change it?